orchestration/hca_orchestration/tests/e2e/test_copy_project.py

```diff
@@ -0,0 +1,164 @@
+                "config": {
+                    "snapshot_steward": "monster-dev@dev.test.firecloud.org"
+                }
+            }
+        }
+    }
+    creation_config = SnapshotCreationConfig(
+        dataset_info.dataset_name,
+        f"{dataset_info.dataset_name}_snapshot_test",
+        False
+    )
+    snapshot_job = cut_snapshot.to_job(
+        resource_defs={
+            "data_repo_client": preconfigure_resource_for_mode(jade_data_repo_client, "dev"),
+            "data_repo_service": data_repo_service,
+            "gcs": google_storage_client,
+            "hca_manage_config": preconfigure_resource_for_mode(hca_manage_config, "dev"),
+            "io_manager": preconfigure_resource_for_mode(gcs_pickle_io_manager, "dev"),
+            "sam_client": preconfigure_resource_for_mode(sam_client, "dev"),
+            "slack": console_slack_client,
+            "snapshot_config": ResourceDefinition.hardcoded_resource(creation_config),
+            "dagit_config": preconfigure_resource_for_mode(dagit_config, "dev"),
+        },
+        executor_def=in_process_executor
+    )
+
+    snapshot_job_result = execute_pipeline(snapshot_job, run_config=snapshot_config)
+    snapshot_info = snapshot_job_result.result_for_solid(
+        "get_completed_snapshot_info").materializations_during_compute[0]
+
+    yield snapshot_info
+
+    # clean up the snapshot when finished
+    logging.info(
+        f"Deleting snapshot, name = {snapshot_info.tags['snapshot_name']}, id = {snapshot_info.tags['snapshot_id']}")
+    response = data_repo_client.delete_snapshot(id=snapshot_info.tags["snapshot_id"])
+    poll_job(response.id, 300, 2, data_repo_client)
+
+
+@pytest.fixture
+def copied_dataset(snapshot, copy_project_config, hca_project_id: str, data_repo_client: RepositoryApi):
+    base_copy_project_config = copy_project_config.copy()
+    base_copy_project_config["resources"]["hca_project_copying_config"] = {
+        "config": {
+            "source_bigquery_project_id": snapshot.tags['data_project'],
+            "source_bigquery_region": "US",
+            "source_hca_project_id": hca_project_id,
+            "source_snapshot_name": snapshot.tags['snapshot_name']
+        }
+    }
+    copy_project_job = copy_project_to_new_dataset_job("dev", "dev")
+    result: PipelineExecutionResult = execute_pipeline(
+        copy_project_job,
+        run_config=base_copy_project_config
+    )
+    copied_dataset = result.result_for_solid("validate_copied_dataset").materializations_during_compute[0]
+
+    yield copied_dataset
+
+    # clean up the copied dataset when finished
+    logging.info(f"Deleting copied dataset, id = {copied_dataset.tags['dataset_id']}")
+    response = data_repo_client.delete_dataset(id=copied_dataset.tags["dataset_id"])
+    poll_job(response.id, 600, 2, data_repo_client)
+
+
+@pytest.mark.e2e
+def test_copy_project(hca_project_id, copied_dataset, tdr_bigquery_client: Client):  # (copied_dataset,
```

Comment:
All of the actual machinery to copy the project is accomplished in the above fixtures; this test is purely assertions against the produced dataset.